forge-gui/res/cardsfolder/upcoming/kaito_dancing_shadow.txt

```diff
@@ -0,0 +1,15 @@
+Name:Kaito, Dancing Shadow
+ManaCost:2 U B
+Types:Legendary Planeswalker Kaito
+Loyalty:3
+T:Mode$ DamageDoneOnce | CombatDamage$ True | ValidSource$ Creature.YouCtrl | TriggerZones$ Battlefield | ValidTarget$ Player | Execute$ TrigChoose | TriggerDescription$ Whenever one or more creatures you control deal combat damage to a player, you may return one of them to its owner's hand. If you do, you may activate loyalty abilities of NICKNAME twice this turn rather than only once.
+SVar:TrigChoose:DB$ ChooseCard | DefinedCards$ TriggeredSources | SubAbility$ DBBounce
+SVar:DBBounce:DB$ ChangeZone | Origin$ Battlefield | Destination$ Hand | Hidden$ True | Mandatory$ True | Defined$ ChosenCard | ChangeNum$ 1 | RememberChanged$ True | SubAbility$ DBLoyalty
+SVar:DBLoyalty:DB$ Effect | StaticAbilities$ PWTwice | ConditionDefined$ Remembered | ConditionPresent$ Card | SubAbility$ DBCleanup | SpellDescription$ You may activate the loyalty abilities of NICKNAME twice this turn rather than only once.
+SVar:PWTwice:Mode$ NumLoyaltyAct | ValidCard$ Card.EffectSource | Twice$ True | Description$ You may activate the loyalty abilities of NICKNAME twice this turn rather than only once.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+A:AB$ Pump | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | TargetMin$ 0 | TargetMax$ 1 | ValidTgts$ Creature | TgtPrompt$ Select up to one target creature | KW$ HIDDEN CARDNAME can't attack or block. | IsCurse$ True | Duration$ UntilYourNextTurn | AILogic$ DetainNonLand | SpellDescription$ Up to one target creature can't attack or block until your next turn.
```

Comment:
`StackDescription$ {c:Targeted} can't attack or block until your next turn.`

(Auto stack description for these hidden keywords is ugly)

Reply:
Thanks for the catch